fix/multiwavefldn.inp.tmpl

```diff
@@ -11,7 +11,7 @@ $
 T
 $
 ww3.
-8
+6
```

Comment:
is it for retro only or operation? I assume we shouldn't change the exisiting format.